guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-server.adoc

```diff
@@ -39,12 +39,11 @@ Do not delete the certificate files after you deploy the certificate.
 They are required when upgrading {ProjectServer}.
 ====
 
-[NOTE]
-====
-If a new consumer package `katello-ca-consumer-latest.noarch.rpm` is generated due to a different Certificate Signing Authority, all the clients registered to {ProjectServer} must be updated.
-====
-
 .Verification
 . Access the {ProjectWebUI} from your local machine.
-For example, https://{foreman-example-com}.
+For example, `https://{foreman-example-com}`.
 . In your browser, view the certificate details to verify the deployed certificate.
+
+.Next steps
+* If you have changed the CA certificate on {ProjectServer}, refresh the CA certificate on your hosts.
```

Comment:
If the user already changed the CA cert, then he needs to refresh the certs manually, and the automatic procedure will probably fail (the hosts won't recognize the Foreman server).

Reply:
But then it doesn't make sense to tell the users that they can access the template over HTTP if they can't use the procedure at all?

Reply:
If the CA is already changed, then most chances are that the server won't be able to execute Remote Execution jobs on the host. This is why only the manual workflow with HTTP endpoint would be possible. As in ssh to the host, download the script from HTTP endpoint and run it.